wordle-web/pages/game.vue

```diff
@@ -1,116 +1,153 @@
-        <v-card loading>
-          <v-card-title class="justify-center">
-            You're being exploited for ad revenue, please standby...
-          </v-card-title>
-          <PrerollAd />
-        </v-card>
-      </v-row>
-    </v-container>
+  <v-container fluid fill-height justify-center>
+    <!-- <v-tooltip bottom>
+      <template #activator="{ on, attrs }">
+        <v-btn color="primary" nuxt to="/" fab v-bind="attrs" v-on="on">
+          <v-icon> mdi-home </v-icon>
+        </v-btn>
+      </template>
+      <span> Go Home </span>
+    </v-tooltip> -->
 
-    <v-container v-if="isLoaded">
-      <v-row justify="center">
-        <v-col cols="1" class="mt-0 mb-0 pt-0 pb-0">
-          <v-tooltip bottom>
-            <template #activator="{ on, attrs }">
-              <v-container>
-                <v-row justify="center">
-                  <v-btn
-                    color="primary"
-                    x-small
-                    nuxt
-                    to="/"
-                    fab
-                    v-bind="attrs"
-                    v-on="on"
-                  >
-                    <v-icon>mdi-home</v-icon>
-                  </v-btn>
-                </v-row>
-              </v-container>
-            </template>
-            <span> Go Home </span>
-          </v-tooltip>
-        </v-col>
-      </v-row>
+    <!-- <v-card-text class="text-h1 font-weight-black text-center">
+      Wordle!
+    </v-card-text> -->
 
-      <v-row justify="center" class="mt-0 pt-2">
-        <v-col class="mt-2 mb-0 pt-0 pb-0">
-          <v-card flat color="transparent" class="mt-0 mb-0 pt-0 pb-0">
-            <v-card-text
-              class="text-h3 font-weight-black text-center ma-0 pa-0"
-            >
-              !Wordle
-            </v-card-text>
-          </v-card>
-        </v-col>
+    <v-alert v-if="wordleGame.gameOver" width="80%" :type="gameResult.type">
+      {{ gameResult.text }}
+      <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+    </v-alert>
+    <v-dialog
+    v-model="leaderboardPrompt"
+    elevation="0"
+    width="40%"
+    :persistent="true">
+      <v-card class = "px-0 py-0 mx-0 my-0 justify-center" min-height="200">
+        <h1>Enter a name to track your score!</h1>
+        <v-divider />
+          <v-text-field v-model="playerName" outlined class ="px-15 my-3" />
+        <v-card-actions>
+          <v-btn @click="leaderboardPrompt=false, submitName()"> Submit Name </v-btn>
+          <v-spacer />
+          <v-btn @click="leaderboardPrompt=false, cancelNameEntry()"> No Thanks! </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+    <v-col>
+      <v-row>
+      Game Time: {{(stopwatch.currentTime/1000/60/60)>1?Math.floor(stopwatch.currentTime/1000/60/60)+":":""}}{{Math.floor(stopwatch.currentTime/1000/60)}}:{{Math.floor(stopwatch.currentTime/1000%60)< 10?'0'+Math.floor(stopwatch.currentTime/1000%60):Math.floor(stopwatch.currentTime/1000%60)}}
```

Comment:
I like the game time.  I implemented time, but did not display it. I like this very much.